modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Classes.js

```diff
@@ -36,13 +36,11 @@ export default (options) => {
                 // If no valid classes for this parse, default to the
                 // the first setting for this tag (including null for tags defined without classes).
                 // else, remove classes.
-                return {
-                  class: classes.length
-                    ? classes.join(' ')
-                    : (
-                      allow[tag].length ? allow[tag][0] : null
-                    )
-                };
+                return classes.length
+                  ? classes.join(' ')
+                  : (
+                    allow[tag].length ? allow[tag][0] : null
+                  );
```

Comment:
unrelated to the goal of this ticket but is the fix for [this one](https://linear.app/apostrophecms/issue/PRO-2091/tiptap-breaking-change).

This break was preventing attributes from being applied to elements parse, which resulted in attributes not being applied on reload

Reply:
Eyyy nice